challenge_5/java/jdfurlan/src/FindTheDifference.java

```diff
@@ -0,0 +1,21 @@
+
+public class FindTheDifference {
+
+	public static void main(String[] args) {
+		String s = "abcd";
+		String t = "dbeca";
+
+		boolean[] b = new boolean[128];
```

Comment:
This wouldn't work if there were multiple of the same character in the given strings. A simple change to using int to hold counts of the letter occurrences would fix that though.

Also, the problem description states that these will only be lowercase letters (ascii assumed) so you only need an array of 26.